### PR TITLE
Jcs/services drop down

### PR DESCRIPTION
**Pull request by jSylvestre**
Example of config setting:
"Services": "fake:jsylvestre@ucdavis.edu,Hide Body:jsylvestre@outlook.edu, Other:jsylvestre@ucdavis.edu"

### CODE REVIEW

**Review by sprucely — CHANGES_REQUESTED** [review]
Didn't test. Looks fine except for that `!=`.

File: src/CaesHelp/ClientApp/src/components/Ticket.tsx [hunk 1]
```diff
@@ -250,11 +263,11 @@ export default class Ticket extends React.Component<
 
   public render() {
     const programmingSupportTitle =
-      '<b>Programming Support:</b> (Scott Kirkland, Ken Taylor, Jason Sylvestre, Spruce Weber-Milne)';
+      '<b>Programming Support:</b> (Scott, Ken, Jason, Spruce)';
```
Comment (laholstege) [inline src/CaesHelp/ClientApp/src/components/Ticket.tsx:266]: should i be in here? :)

**Review by laholstege — APPROVED** [review]
didnt test but looks good

File: src/CaesHelp/ClientApp/src/components/Ticket.tsx (hunk 1 above)
Comment (jSylvestre) [reply src/CaesHelp/ClientApp/src/components/Ticket.tsx:266]: @laholstege Only if you have access to Service Now to answer help tickets.

Comment (jSylvestre) [reply src/CaesHelp/ClientApp/src/components/Ticket.tsx:266]: Only if you have access to Service Now to answer help tickets

Comment (laholstege) [reply src/CaesHelp/ClientApp/src/components/Ticket.tsx:266]: cool, i don't and i'm happy with that

**Pull request merged**